client/my-sites/site-settings/form-seo.jsx

```diff
@@ -36,6 +36,9 @@ const serviceIds = {
 	yandex: 'yandex-verification'
 };
 
+// Matches any open or closed html tag
```

Comment:
If it were me, I would admit that this can fail but that it's okay

```js
// Matches any open or closed html tag
// can detect false-positives but that
// won't hurt us her
```